src/ducks/Signals/signalFormManager/signalCrudForm/signal/TriggerForm.js

```diff
@@ -131,6 +141,25 @@ export const TriggerForm = ({
               ) {
                 setDefaultPriceValues(current.values)
                 validateForm()
+                return
+              }
+
+              if (!isEqual(current.values, prev.values)) {
+                const lastErrors = validateTriggerForm(current.values)
+                const isError = Object.keys(current.values).reduce(
+                  (acc, val) => {
+                    if (lastErrors.hasOwnProperty(val)) {
+                      acc = true
+                    }
+                    return acc
+                  },
+                  false
+                )
+
+                !!current.values.target &&
```

Comment:
If the value is `truthy`, then `!!` will just convert it to `true` and that's not necessary